spec/fdoc/endpoint_scaffold_spec.rb

```diff
@@ -113,7 +113,7 @@
     it "produces a valid JSON schema for the response" do
       subject.consume_request(request_params)
       subject.request_parameters["properties"].should have(3).keys
-      JSON::Validator.validate!(subject.request_parameters, request_params, :version => :draft3).should be_true
+      JSON::Validator.validate!(subject.request_parameters, request_params, :version => :draft3).should be_truthy
```

Comment:
Could this and all the other changed occurrences be `be(true)` instead? `truthy` has a slightly different meaning